atlp-rwanda/ecommerce-app-predators-fe
# Fix: Fixing an unsuccessful rebase

#### What does this PR do?
- Reverts develop branch to the 2nd last merge.

#### Description of Task to be completed?
- Fixing develop to recover lost codes.

#### How should this be manually tested?
```
git clone https://github.com/atlp-rwanda/ecommerce-app-predators-fe.git
cd ecommerce-app-predators-fe
git checkout fx-merge-error
npm i
npm run dev
```
- Check if some necessary features are there!
